deepchem/models/tensorgraph/layers.py

```diff
@@ -983,6 +989,22 @@ def create_tensor(self, in_layers=None, set_tensors=True, **kwargs):
       self.out_tensor = out_tensor
     return out_tensor
 
+def geqk(x,k):
```

Comment:
Where should I add this? I think I should add it to GraphGather, would that be ok?